### PR TITLE
Add test cases

### PR DESCRIPTION
**Pull request by morsh**

### CODE REVIEW

File: client/src/tests/elements/Pie.test.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          props, 
+          title, 
+          subtitle);
+        pie = TestUtils.renderIntoDocument(pieElement);
+        TestUtils.isElementOfType(pie, 'div');
+      }
+      catch (e) {
+        console.error(e);
+      }
+      setTimeout(done, 100);
```
Comment (eladiw) [inline client/src/tests/elements/Pie.test.tsx]: In case of time out , shouldnt you call 'throw' and then 'done'?

**Comment by morsh** [comment]
@eladiw - I added the following fix:

```ts
      try {
        ///...

        // Adding a timeout to make sure Flux cycle is complete
        setTimeout(done, 100);
        
      } catch (e) {
        return done(e);
      }
```

This way it's clearer what the setTimeout is for

File: client/src/tests/elements/Pie.test.tsx [hunk 2]
```diff
@@ -40,11 +39,13 @@ describe('Pie', () => {
           subtitle);
         pie = TestUtils.renderIntoDocument(pieElement);
         TestUtils.isElementOfType(pie, 'div');
+
+        // Adding a timeout to make sure Flux cycle is complete
+        setTimeout(done, 100);
+        
+      } catch (e) {
+        return done(e);
```
Comment (eladiw) [inline client/src/tests/elements/Pie.test.tsx]: what about failing the test here and also within the settimout (if i got it right, those are the negative flows)

Comment (morsh) [inline client/src/tests/elements/Pie.test.tsx]: That's what `done` does. first parameter is error

**Pull request merged**